Source/Scene/Camera.js

```diff
@@ -1687,6 +1698,9 @@ define([
         Matrix3.multiplyByVector(rotation, direction, direction);
         Matrix3.multiplyByVector(rotation, up, up);
         Matrix3.multiplyByVector(rotation, right, right);
+
+        _limitOrientation(angle, this);
```

Comment:
I'm not sure this is doing the right thing.  `angle` is the angle to move the camera by, and is not directly related to the heading/pitch/roll the camera is limited to